jest.config.js

```diff
@@ -0,0 +1,12 @@
+const testsForPackage = (packageName) => `<rootDir>/packages/${packageName}/**/*.test.[jt]s`
+
+module.exports = {
+  projects: [
+    {
+      displayName: 'core packages',
+      testMatch: [
+        testsForPackage('core')
```

Comment:
Beyond the different different environment requirements for some tests (browser/node/react-native) the grouping is fairly arbitrary. We could add more groups for structure. This will shape up some more as we add more jest coverage.